src/ResourceManagement/AppService/AppServiceBaseImpl.cs

```diff
@@ -185,6 +185,11 @@ public FluentImplT WithFreePricingTier()
             return WithPricingTier(new PricingTier("Free", "F1"));
         }
 
+        public FluentImplT WithConsumptionPricingTier()
```

Comment:
Not sure where the `///GENMHASH` on other methods come from... Is it supposed to be generated automatically?

Reply:
HASH is not needed.

In the implementation of this method, it should be able to use `ConsumptionY1` directly?

Reply:
> In the implementation of this method, it should be able to use `ConsumptionY1` directly?

Yes, I thought about it, but I saw that the other methods didn't use the predefined pricing tiers, so I didn't either, for consistency. I can change it if you like.

Reply:
Yes, please.

Reply:
Done (I amended the commit)